contracts/FlowIDTableStaking.cdc

```diff
@@ -1191,11 +1199,15 @@ pub contract FlowIDTableStaking {
         pre {
             FlowIDTableStaking.stakingEnabled(): "Cannot register a node operator if the staking auction isn't in progress"
         }
-        let newNode <- create NodeRecord(id: id, role: role, networkingAddress: networkingAddress, networkingKey: networkingKey, stakingKey: stakingKey, tokensCommitted: <-tokensCommitted)
+        let newNode <- create NodeRecord(id: id, role: role, networkingAddress: networkingAddress, networkingKey: networkingKey, stakingKey: stakingKey, tokensCommitted: <-FlowToken.createEmptyVault())
```

Comment:
Isn't it confusing here that in the `NodeRecord` we are committing an empty vault while at line 1208 we are emitting `TokensCommitted` event with a non-zero value?

Reply:
I probably should have included this change in [the next PR](https://github.com/onflow/flow-core-contracts/pull/324) because it relates to the candidate node list. I didn't want to have duplicate code for adding it to the candidate node list, so I just created an empty vault here instead and committed tokens the regular way. Does that make sense?